pkg/meta/meta.go

```diff
@@ -90,6 +90,15 @@ func (env *Environment) Profile() *localdata.Profile {
 	return env.profile
 }
 
+// Close release resource of env.
+func (env *Environment) Close() error {
+	if os.Getenv(EnvNameV1) == "" {
```

Comment:
I think this check should be `env.repo != nil` (once the repos are initialized we should no longer check the env vars in order to make them easy to override, e.g., in testing).